setup.py

```diff
@@ -28,20 +29,18 @@
               file=sys.stderr)
         SOURCE_EXT = 'c'
 
-get_output = None
 
-try:
-    import commands
-    get_output = commands.getoutput
-except ImportError:
-    import subprocess
+try:  # TODO: Remove this when Py27 is dropped
```

Comment:
Move to just below the imports.

Reply:
After thinking about it again, I decided to just use OSError instead of FileNotFoundError as it's close enough and works without the hack.